src/Form.svelte

```diff
@@ -22,6 +39,22 @@
   export let disabled = false;
   export let config: Config;
 
+  let formFields = fields;
+
+  const check = (e: any) => {
+    formFields = formFields.map(field => {
+      if (field.name === e?.target?.name) {
+        const hasErrors = field.validate && !validationTypes[field.validate].test(e?.target?.value);
+        submitDisabled = hasErrors ? true : false;
+        return { ...field,
+          value: e?.target?.value,
+          error: hasErrors ? `Invalid input, expected ${field.validate}` : undefined
```

Comment:
Since the user will get this error before they are done typing, perhaps we should phrase it a bit more softly, eg. `Must be a valid ${field.validate}`

Reply:
Yeah no problem :+1: